client/etcd/options.go

```diff
@@ -35,14 +35,14 @@ type Options interface {
 	Zone() string
 	SetZone(z string) Options
 
-	ServiceInitTimeout() time.Duration
-	SetServiceInitTimeout(timeout time.Duration) Options
+	Service() string
```

Comment:
Yeah I thought about that, I didn't do that because user of this Option does not necessarily need to use the service discovery client which contains the idea of ServiceID, they can just use the KV interface, also the validation for this Option only requires the Service field to be non-empty, while ServiceID has different validation rules so I can't call serviceID.Validate() here but have to call `opts.ServiceID().Service() != ""` which is a bit weird